scene/gui/code_edit.cpp

```diff
@@ -1291,15 +1291,22 @@ bool CodeEdit::is_line_numbers_zero_padded() const {
 
 void CodeEdit::_line_number_draw_callback(int p_line, int p_gutter, const Rect2 &p_region) {
 	String fc = TS->format_number(String::num(p_line + 1).lpad(line_number_digits, line_number_padding));
-	Ref<TextLine> tl;
-	tl.instantiate();
-	tl->add_string(fc, font, font_size);
-	int yofs = p_region.position.y + (get_line_height() - tl->get_size().y) / 2;
+	TextLine tl;
+	tl.add_string(fc, font, font_size);
+
+	if (p_line == 0) {
```

Comment:
Should check for `get_first_visible_line() - 1` rather then `0`, else it'll only run if line `0` is visible. Not sure we can guarantee that if `go_to_line` is used.